src/lib/stores/app-state-store.ts

```diff
@@ -20,7 +20,7 @@ interface AppState {
   coachingRelationship: CoachingRelationshipWithUserNames;
 }
 
-interface AppStateActions {
+export interface AppStateActions {
```

Comment:
```suggestion
interface AppStateActions {
```
@qafui this is meant to be private to AppStateStore, any reason to broaden its scope?